codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/CodegenVisitor.kt

```diff
@@ -229,12 +239,28 @@ class CodegenVisitor(context: PluginContext, private val codegenDecorator: RustC
         }
     }
 
+    /**
+     * Union Shape Visitor
+     *
+     * Generate an `enum` for union shapes.
+     *
+     * Note: this does not generate serializers
```

Comment:
```suggestion
     * This function _does not_ generate any serializers.
```

Consistent with the above.